src/Custom/Assistants/MessageContent.cs

```diff
@@ -19,7 +19,7 @@ public static MessageContent FromImageFileId(
             => new InternalMessageContentImageFileObject(
                 imageFile: new(
                     fileId: imageFileId,
-                    detail: detail?.ToString(),
+                    detail: detail?.ToString().ToLowerInvariant(),
```

Comment:
Instead of calling `ToString().ToLowerInvariant`, it would be more appropriate to call the `ToSerialString()` extension method, so that this works regardless of renames, snake casing, or any other transformations (of which fortunately there are none today in this particular case... but just in case this changes in the future 🙂).

Reply:
Thanks for pointing that out. I wasn’t aware of the `ToSerialString()` extension method. I've updated the code to use it.